ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.12-presubmits.yaml

```diff
@@ -1873,7 +1873,7 @@ presubmits:
     branches:
     - ^release-4\.12$
     - ^release-4\.12-
-    cluster: build08
+    cluster: build05
```

Comment:
This looks odd.  It's usually signal that you should rebase your PR and re-run `make update`.

Reply:
It was exactly in opposite way: I added `reporter_config:` and `ci/prow/ordered-prow-config` started failing: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43396/pull-ci-openshift-release-master-ordered-prow-config/1704931832371351552

To workaround these weird failures, I rebased my PR, re-run `make update`, and the latter did `s/build08/build05`. I agree this looks odd, yes!